app/src/main/java/org/glucosio/android/fragment/OverviewFragment.java

```diff
@@ -281,15 +278,15 @@ public void onNothingSelected(AdapterView<?> parent) {
         xAxis.setTextColor(getResources().getColor(R.color.glucosio_text_light));
         xAxis.setAvoidFirstLastClipping(true);
 
-        int minGlucoseValue = presenter.getGlucoseMinValue();
-        int maxGlucoseValue = presenter.getGlucoseMaxValue();
+        double minGlucoseValue = presenter.getGlucoseMinValue();
+        double maxGlucoseValue = presenter.getGlucoseMaxValue();
 
         LimitLine ll1;
         LimitLine ll2;
 
-        if (("mg/dL").equals(presenter.getUnitMeasuerement())) {
-            ll1 = new LimitLine(minGlucoseValue);
-            ll2 = new LimitLine(maxGlucoseValue);
+        if (DomainConstants.MG_D_L.equals(presenter.getUnitMeasurement())) {
+            ll1 = new LimitLine((float) minGlucoseValue);
```

Comment:
Why in the `if` the `ll1` holds `minGlucoseValue` and `ll2` holds `maxGlucoseValue` and in the `else` is opposite?